skipper.go

```diff
@@ -1505,6 +1505,10 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		o.CustomFilters = append(o.CustomFilters, compress)
 	}
 
+	if ratelimitRegistry != nil && redisOptions != nil {
+		o.CustomFilters = append(o.CustomFilters, ratelimitfilters.NewLeakyBucket(ratelimitRegistry))
```

Comment:
why not adding it close to the other ratelimit settings?
I see it's redis specific, but you can omit the first condition and just check the latter, if you put it close to the others.